logging/tests/system.py

```diff
@@ -171,6 +171,24 @@ def test_log_text_with_timestamp(self):
         self.assertEqual(entries[0].payload, text_payload)
         self.assertEqual(entries[0].timestamp, now.replace(tzinfo=UTC))
 
+    def test_log_text_with_resource(self):
```

Comment:
It works for me on both http and gax but I don't believe system tests run on both. Seems parametizing system tests for both is higher level issue?